packages/rps/src/redux/message-service/saga.ts

```diff
@@ -27,87 +28,48 @@ export default function* messageSaga() {
   yield fork(sendWalletMessageSaga);
   yield fork(receiveChallengePositionFromWalletSaga);
   yield fork(receiveChallengeFromWalletSaga);
-  yield fork(recieveDisplayEventFromWalletSaga);
+  yield fork(receiveDisplayEventFromWalletSaga);
   yield fork(exitGameSaga);
 }
 
-interface CommitmentMessage {
-  data: Commitment;
+interface AppMessage {
+  commitment: Commitment;
   signature: string;
-  queue: Queue;
+  queue: Queue.GAME_ENGINE;
   userName?: string;
 }
 
-interface NonCommitmentMessage {
-  data: string;
+interface WalletMessage {
```

Comment:
It's potentially confusing that this is a different `WalletMessage` from the one defined above. Maybe the other one should be `WalletMessagePayload` or similar.

Reply:
Good point, done!